Samples/AppLifecycle/Activation/cpp-winui-packaged/CppWinUiDesktopActivation/CppWinUiDesktopActivation (Package)/Package.appxmanifest

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap5="http://schemas.microsoft.com/appx/manifest/uap/windows10/5"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="60706fc7-b419-4ce6-9a8a-4f99a62fd020"
+    Publisher="CN=andreww"
```

Comment:
We've been using "Microsoft Corporation" for other samples: https://github.com/microsoft/WindowsAppSDK-Samples/blob/81f0e7327e2bf1dacf8c4d0e752aa0e600abf18b/Samples/ResourceManagement/cs-winui/winui_desktop_packaged_app%20(Package)/Package.appxmanifest#L11

Not sure what the official guidance should be here, but change to Microsoft for consistency?